docs/topics/rn-new-features.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/release_notes/master.adoc
 
+:_content-type: REFERENCE
 [id="rn-new-features_{context}"]
 = New features
 
```

Comment:
@stoobie @sbeskin-redhat I'd call this a content module. It gives a brief description of new features, not the technica details you'd expect in a reference module.

Reply:
@sbeskin-redhat @RichardHoch 
I'd call this a CONCEPT. It's a series of paragraphs, and the info is meant to be read once and not referred to again.

Also, the first sentence is self-referential, which we are supposed to avoid:
This section describes the new features of the Migration Toolkit for Applications (MTA) 5.2.1. 

I think you can just delete the sentence, or what would be useful is a summary of the new features, e.g.
```suggestion
New features include:
* Rules for migrating from Java EE8 to Jakarta EE9
* OpenRewrite recipe support
* MTA extension for Visual Studio Code compatible with Codespaces
* Rules for the Springboot-to-Quarkus migration path
```
And then the following paragraphs provide more info on each one.